src/components/user_view/UserInfoContainer.js

```diff
@@ -4,14 +4,16 @@ import { updateUserInfoInDb } from '../../redux/actions/userActions';
 
 const mapStateToProps = (state) => {
   return {
-    f_name: state.user.user.f_name,
-    l_name: state.user.user.l_name,
-    email: state.user.user.email
+    user: state.user.user
```

Comment:
I don't really have a preference either way, but out of curiosity, why did you change from the three props to the one? On one hand, we don't have to add any props if we decide this needs access to more info. On the other, components could be given access to more than they should be allowed to. This might not be a big deal though. I don't think we've been following any one convention so far, now is probably a good time to decide on how we want to standardize it :)

Reply:
Good point - I would prefer to use the three props rather than the entire object. I think it's a more verbose and consequently better convention to continue following. Will fix that